### PR TITLE
About page

### PR DESCRIPTION
**Pull request by Adouleur**

### CODE REVIEW

File: app/components/layout/Email.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+const Email = ({ className, fill, fillOpacity }: Props) => {
+  return (
+    <div className={` flex items-center ${className}`}>
+      <SvgEmailHeader
+        className="mr-1.5"
+        fill={fill}
+        fillOpacity={fillOpacity}
+      />
+      <p>request@example.com</p>
```
Comment (MiradorOne) [inline app/components/layout/Email.tsx]: should be `<a href="mailto:request@example.com"`

File: app/components/layout/Phone.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+}
+const Phone = ({ className, fill, fillOpacity }: Props) => {
+  return (
+    <div className={` flex items-center ${className}`}>
+      <SvgWhatsAppHeader
+        className="mr-1.5"
+        fill={fill}
+        fillOpacity={fillOpacity}
+      />
+      <p>+971 55 811 9024</p>
```
Comment (MiradorOne) [inline app/components/layout/Phone.tsx]: `<a href=“tel:555-666-7777”`

**Review by MiradorOne — CHANGES_REQUESTED** [review]
small fixes and g2g

**Comment by Adouleur** [comment]
Done!

**Comment by MiradorOne** [comment]
@Adouleur conflicts

**Comment by Adouleur** [comment]
Resolved, please check @MiradorOne

**Pull request merged**